docopt.go

```diff
@@ -181,7 +181,15 @@ func parse(doc string, argv []string, help bool, version string, optionsFirst bo
 		return
 	}
 
-	err = newUserError("")
+	if len(*left) > 0 {
+		var left_names []string
+		for _, l := range(*left) {
+			left_names = append(left_names, l.name)
+		}
+		err = newUserError(fmt.Sprintf("Unknown option(s): %s", strings.Join(left_names, ", ")))
```

Comment:
@caldwell - I believe `golint` will complain about this error message because the first letter is capitalized. 
https://github.com/golang/go/wiki/CodeReviewComments#error-strings